modules/testPortlet/src/main/resources/META-INF/resources/view.jsp

```diff
@@ -5,10 +5,23 @@
         <portlet:param name="javax.portlet.action" value="addNotice"/>
     </portlet:actionURL>
 
-    <c:forEach items="${userNotice}" var="noticeLine" varStatus="status">
-        <p>${noticeLine}</p>
+    <portlet:actionURL var ="deleteNotice">
+        <portlet:param name="javax.portlet.action" value="deleteNoticeById"/>
+    </portlet:actionURL>
+
```

Comment:
it is incorrect implementation, we have spoken about input textarea, where you can update you notice, change it